static/django_templates/base/index.html

```diff
@@ -1,4 +1,4 @@
-{% load staticfiles %}
+{% load staticfiles compress %}
```

Comment:
For it to work you either have to set `APP_DIRS: True` or specify the `loaders` app_directories loader to be exact